README.md

```diff
@@ -60,14 +52,101 @@ Spinal cord MRI data:
 - T2star tra
 Brain MRI data:
 - DWI (A-P and P-A phase encoding)
-- MPM (multi-parameter mapping)
+- T1w
 
 ## naming convention
 
 sub-<site><pathology>XXX
 
 example:
-sub-zurichDCM001
+sub-montrealDCM001
+
+<ADDITIONAL NOTES IF AVAILABLE>Note: the label `bp-cspine` is used to differentiate spine images from brain.
+
+## derivatives
+
+The derivatives will be organized according to the following:
+
+https://github.com/ivadomed/ivadomed/wiki/repositories#derivatives
+
+Convention for derivatives JSON metadata:
+
+{
+  "Author": "Firstname Lastname",
+  "Date": "YYYY-MM-DD HH:MM:SS"
+}
+
+## <IF DATA ARE MISSING FOR SOME SUBJECT(S)>missing data
+
+```
+
+</details>
+
+## `dataset_description.json`
+
+The `dataset_description.json` is a JSON file describing the dataset.
+
+<details><summary>dataset_description.json template:</summary>
+
+```json
+{
+    "BIDSVersion": "BIDS 1.8.0",
```

Comment:
might need to be updated by now

Reply:
Good catch! Updated: https://github.com/neuropoly/data-management/pull/189/commits/9f8de23d5aedf522798c18db0565dfe92fbec003